commit f67449fdba3b9dbdd340d8cbf17dfa711d5bd2fb
Author: Hans Verkuil <hans.verkuil@cisco.com>
Date:   Wed Mar 21 13:28:47 2018 -0400

    media: debugfs-cec-error-inj: document CEC error inj debugfs ABI
    
    Document the core of the debugfs CEC error injection ABI.
    
    The driver specific commands are documented elsewhere and
    this file points to that documentation.
    
    Signed-off-by: Hans Verkuil <hans.verkuil@cisco.com>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index 2fce65318d4f..1d1ec49cff90 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -3307,6 +3307,7 @@ F:	include/media/cec-notifier.h
 F:	include/uapi/linux/cec.h
 F:	include/uapi/linux/cec-funcs.h
 F:	Documentation/devicetree/bindings/media/cec.txt
+F:	Documentation/ABI/testing/debugfs-cec-error-inj
 
 CEC GPIO DRIVER
 M:	Hans Verkuil <hans.verkuil@cisco.com>